aten/src/ATen/function_wrapper.py

```diff
@@ -489,8 +515,17 @@ def process_option(option, output_options):
         option['method_prefix_derived'] = '' if broadcast_arg is None else 's_'
         env = nested_dict(option, top_env)
 
+        mode = option['mode']
         abstract = True
-        if broadcast_arg is None:
+        if mode == 'NN' and option.get('cimpls') is None:
```

Comment:
can you add a comment describing the cases when mode == 'NN' and 'cimpls' is None vs not-None?  I don't quite follow when each code path is triggered.